src/components/language-flag-toggle-list-item/LanguageFlagToggleListItem.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState } from "react";
+import { IconDragable } from "../..";
+import { SLanguageFlagToggleListItem } from "./styles/SLanguageFlagToggleListItem";
+import { Flag, Toggle } from "..";
+import { ICountryCode } from "../flag/IFlag";
+import { SLanguageFlagToggleListItemDragIcon } from "./styles/SLanguageFlagToggleListItemDragIcon";
+import { SLanguageFlagToggleListItemToggleButton } from "./styles/SLanguageFlagToggleListItemToggleButton";
+
+export interface ILanguageFlagToggleListItemLocale {
+  id: ICountryCode;
+  name: string;
+  active: boolean;
+}
+
+interface ILanguageFlagToggleListItem {
+  locale: ILanguageFlagToggleListItemLocale;
+  onLocaleToggle: (newValue: boolean) => void;
+}
```

Comment:
@nicolasshiken since the element is draggable you me need to pass the drag event handlers as well.